src/js/components/UserInformation/Address.scss

```diff
@@ -0,0 +1,3 @@
+.addr {
+  word-wrap: break-word;
```

Comment:
Boot strap has a class for that: https://embark.status.im/docs/contracts_configuration.html#Providing-ABIs